js_modules/dagster-ui/packages/ui-core/src/metadata/MetadataEntry.tsx

```diff
@@ -73,10 +81,12 @@ export const MetadataEntries = ({
   }
   return (
     <LogRowStructuredContentTable
-      rows={entries.map((entry) => ({
-        label: entry.label,
-        item: <MetadataEntry entry={entry} expandSmallValues={expandSmallValues} />,
-      }))}
+      rows={entries
+        .filter((entry) => !HIDDEN_METADATA_ENTRY_LABELS.has(entry.label))
```

Comment:
Hmmmm I also changed this in my PR to hide `lineage` if it is a JSONMetadataEntry. I don't think that impacts this, but there would be a second filter case here. 

At some point it could be worth adding a `hidden:` attribute to the metadata that we set on the backend to control this...